### PR TITLE
[REF] getaddons: Fix 'unknown revision' error in get modules changed

### PR DESCRIPTION
**Pull request by moylop260**
Fix OCA/maintainer-quality-tools#277

### CODE REVIEW

File: travis/getaddons.py [hunk 1]
```diff
@@ -76,7 +76,12 @@ def get_modules_changed(path, ref='HEAD'):
     :return: List of paths of modules changed
     '''
     git_run_obj = GitRun(os.path.join(path, '.git'))
-    git_run_obj.run(['fetch'] + ref.split('/'))
+    if ref != 'HEAD':
+        fetch_ref = ref
+        if ':' not in fetch_ref:
+            # to force create branch
+            fetch_ref += ':' + fetch_ref
```
Comment (pedrobaeza) [inline travis/getaddons.py]: You duplicate ref here?

Comment (moylop260) [inline travis/getaddons.py]: Yes,
We need create branch with fetch

The final command is: 
```python
ref = origin/8.0
git fetch origin 8.0:origin/8.0
```

**Comment by moylop260** [comment]
FYI tested in next dummy PR: https://github.com/OCA/partner-contact/pull/188
Build: https://travis-ci.org/OCA/partner-contact/jobs/85573030
Result: green

**Comment by pedrobaeza** [comment]
OK then :+1:

Thanks!

**Comment by moylop260** [comment]
IMHO this PR needs a fast-tracking to avoid errors in build in project with branch default 9.0

**Comment by pedrobaeza** [comment]
Yeah, done!

**Comment by moylop260** [comment]
Thanks @pedrobaeza

**Comment by dreispt** [comment]
Thanks for the quick reaction!

**Comment by moylop260** [comment]
You are welcome @dreispt  for that we are

**Pull request merged**